lib/inferno/apps/cli/main.rb

```diff
@@ -10,6 +11,47 @@
 module Inferno
   module CLI
     class Main < Thor
+      desc 'evaluate', 'Run a FHIR Data Evaluator.'
+      long_desc <<-LONGDESC
+        Evaluate FHIR data in the context of a given Implementation Guide,
+        by applying a set of predefined rules designed to check that datasets are comprehensive.
+        Issues identified will be printed to console or to a json file.
+
+        You must have background services running: `bundle exec inferno services start`
+
+        Run the evaluation CLI with
+
+        `bundle exec inferno evaluate ig_path [data_path]`
+
+        Examples:
+
+        # Load the us core ig and evaluate the data in the provided example folder. If there are examples in the IG already, they will be ignored.
+        `bundle exec inferno evaluate ./uscore.tgz ./package/example`
```

Comment:
I think these examples don't match the actual command to run as currently implemented. Since we're saying for now "always require the user to specify an IG" the position arguments would be ok, or making them options like `-i` or `--ig` is ok too. Just need to make sure the impl and documentation are in sync

Reply:
I would generally prefer no more than one positional argument.

Reply:
So IG_PATH as positional argument and DATA_PATH as argument with value?

`bundle exec inferno evaluate uscore7.0.0.tgz -d patient_example.json `

Reply:
Modified